roles/confluent.variables/defaults/main.yml

```diff
@@ -415,10 +433,10 @@ kafka_rest_monitoring_interceptors_enabled: "{{ monitoring_interceptors_enabled
 
 # Kafka Connect Variables
 
-### Only use to customize Linux User Connect Service runs with. User must exist on host.
+### Only use to customize Linux User Connect Service runs with. Default user is cp-kafka-connect
```

Comment:
Modify this variable to customize the Linux User that the Kafka Connect Service runs with. Default user is cp-kafka-connect.